content/advanced/330_servicemesh_using_appmesh/integrate_observability/cloudwatch_logs.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Cloudwatch Logs"
+date: 2020-01-27T08:30:11-07:00
+weight: 92
+draft: false
+---
+
+#### Cloudwatch Console
+
+Logs are collected by the fluentd daemonset running in the EKS nodes. The following CloudWatch log groups are created by default when Container Insights is setup:
+
+* /aws/containerinsights/cluster-name/application
+* /aws/containerinsights/cluster-name/dataplane
+* /aws/containerinsights/cluster-name/host
+* /aws/containerinsights/cluster-name/performance
+
+Go to console, go to Cloudwatch, and go to `LogGroups`, you should see below log groups
```

Comment:
This could be a little smoother, see comment on container_insight.md

Reply:
updated with details